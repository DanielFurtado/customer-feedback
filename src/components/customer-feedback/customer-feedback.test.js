import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import CustomerFeedback from './customer-feedback';

describe(('Customer Feedback component'), () => {
  const feedback = {
    id: 123,
    name: 'Name',
    email: 'test@email.com',
    rating: 5,
    comment: 'this is a comment'
  };

  it('renders', () => {
    const component = renderer.create(
      <CustomerFeedback 
        key={feedback.id} 
        feedback={feedback} 
      />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});