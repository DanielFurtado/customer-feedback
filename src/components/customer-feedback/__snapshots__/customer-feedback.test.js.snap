// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Customer Feedback component renders 1`] = `
Array [
  <div
    className="sc-bcXHqe fCRhLI"
  >
    test@email.com
  </div>,
  <div
    className="sc-gswNZR fYHcfJ"
  >
    this is a comment
  </div>,
  <div
    className="sc-dkrFOg bUNOjm"
  >
    <hr />
  </div>,
]
`;
